### PR TITLE
Website: Complete transition to ERC

### PR DESCRIPTION
**Pull request by Pandapip1**

### CODE REVIEW

File: _layouts/eip.html [hunk 1]
```diff
@@ -113,7 +113,11 @@ <h2>Citation</h2>
...
   https://ieee-dataport.org/sites/default/files/analysis/27/IEEE%20Citation%20Guidelines.pdf
   {% endcomment %}
+  {% if page.category == "ERC" %}
+  <p>{% include authorlist.html authors=page.author %}, "ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status == "Stagnant" or page.status == "Withdrawn" or page.status == "Review" or page.status == "Last Call" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
+  {% elsif page.category != "ERC" %}
   <p>{% include authorlist.html authors=page.author %}, "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status == "Stagnant" or page.status == "Withdrawn" or page.status == "Review" or page.status == "Last Call" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
+  {% endif %}
```
Comment (SamWilsn) [inline _layouts/eip.html:120]: Was more changed, or could we do:

```suggestion
  <p>{% include authorlist.html authors=page.author %}, "{% if page.category == "ERC" %}ERC{% else %}EIP{% endif %}-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status == "Stagnant" or page.status == "Withdrawn" or page.status == "Review" or page.status == "Last Call" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
```

File: _layouts/eip.html [hunk 2]
```diff
@@ -123,9 +123,14 @@ <h2>Citation</h2>
   {
     "@context": "http://schema.org",
     "@type": "TechArticle",
+    {% if page.category == "ERC" %}
+    "headline": "ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status == "Stagnant" or page.status == "Withdrawn" or page.status == "Review" or page.status == "Last Call" %} [DRAFT]{% endif %}",
+    "name": "ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status == "Stagnant" or page.status == "Withdrawn" or page.status == "Review" or page.status == "Last Call" %} [DRAFT]{% endif %}",
+    {% elsif page.category != "ERC" %}
```
Comment (SamWilsn) [inline _layouts/eip.html:129]:
```suggestion
    {% else %}
```

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Comment by Pandapip1** [comment]
@eth-bot rerun

**Review by SamWilsn — APPROVED** [review]

**Comment by derik1984** [comment]
Default

**Pull request merged**